src/data/pnat/basic.lean

```diff
@@ -349,4 +350,126 @@ theorem gcd_mul_lcm (n m : ℕ+) : (gcd n m) * (lcm n m) = n * m :=
 
 def prime (p : ℕ+) : Prop := (p : ℕ).prime
 
+section coprime_and_more_facts
```

Comment:
The `section` is unnecessary if you're not using `variables` or `open`.

You can add headings in module docs like this instead, and they'll show up in the online mathlib_docs:
```suggestion
### coprime
```

I guess this file doesn't have any others yet, but we can do a bigger cleanup of this file later.